examples/SampleApp/ios/SampleApp.xcodeproj/project.pbxproj

```diff
@@ -694,6 +695,8 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
```

Comment:
should be able to avoid changing this and cover it in the fastfile:
```
platform :ios do
  desc "Build app and send to firebase for QA testing"
  before_all do
    setup_ci
  end

  lane :beta do |options|

    # Fetch and Create Profiles
    ["adhoc"].each do |type|
      match(
          app_identifier: app_id,
          readonly: true,
          skip_docs: true,
          type: type
      )

      # Disable and enable from these steps https://docs.fastlane.tools/codesigning/getting-started/#using-xcodes-code-signing-feature
      update_code_signing_settings(path: "path/to/SampleApp.xcodeproj", use_automatic_signing: false)

      # Builds app with Xcode
      build_app()

      #update_code_signing_settings(path: "path/to/SampleApp.xcodeproj", use_automatic_signing: true)
    end
    

    # Uploads to firebase
    firebase_app_distribution(
      app: "our_app_id",
      firebase_cli_token: (options[:cli_token]),
      groups: (options[:silent_build] ? "" : "qa"),
      ipa_path: (options[:skip_build] ? "./build/SampleApp.ipa" : nil),
    )
    end
  end
end
```